src/test/operate.test.js

```diff
@@ -0,0 +1,37 @@
+import operate from '../logic/operate';
+
+test('operate function exists', () => {
```

Comment:
Remove "operate" from all the test description. It is already obvious you testing the operate function.